### PR TITLE
add test condition

### PR DESCRIPTION
**Pull request by DOnEbrala**
test condition

### CODE REVIEW

File: page_objects/PageCondition.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Page = require('./page');
+
+class condition extends Page {
```
Comment (rimus) [inline page_objects/PageCondition.js]: Naming: **C**ondition.

File: page_objects/check.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+class Check extends Page {
+    get checkCondition(){
+        return $(`//ol[@class = "text"]`)
+    }
+
+
+}
+
+module.exports = new Check
```
Comment (rimus) [inline page_objects/check.js]: `new Check();`

File: page_objects/PageCondition.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    get button(){
+        return $('button.btn');
+    }
+    async pasteBinFrame(args , title) {
+        await this.pasteFrame.setValue(args)
+        await this.clickDrop.click();
+        await this.chooseDrop.click();
+        await this.titleFrame.setValue(title);
+        await this.button.click()
+    }
```
Comment (rimus) [inline page_objects/PageCondition.js:33]: Need to add syntax highlighting steps.

File: test/specs/test.spec.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    it("First Task", async () => {
+        await browser.url("https://pastebin.com")
+        await condition.pasteBinFrame("Hello From Webdriver" , "helloweb");
+    });
+    it("Second Task", async () => {
+        await browser.url("https://pastebin.com")
+        await condition.pasteBinFrame('git config --global user.name "New Sheriff in Town"'+ "\uE007" + 'git reset $ (git commit-tree HEAD ^ {tree} -m "Legacy code")'+ "\uE007" +'git push origin master --force' , `how to gain dominance among developers` );
+    });
+    it("Check", async ()=>{
+        await browser.pause(3000)
```
Comment (rimus) [inline test/specs/test.spec.js]: Please replace `browser.pause()`  with waiters (e.g. `waitForExist()`) → https://webdriver.io/docs/api/element/waitForExist

File: test/specs/test.spec.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        await expect(browser).toHaveTitleContaining("how to gain dominance among developers");
+        await expect(check.checkCondition).toHaveTextContaining('git config --global user.name "New Sheriff in Town"','git reset $ (git commit-tree HEAD ^ {tree} -m "Legacy code")','git push origin master --force');
+    })
+});
```
Comment (rimus) [inline test/specs/test.spec.js:19]: Please check Bash on the page.

**Review by rimus — CHANGES_REQUESTED** [review]

**Comment by DOnEbrala** [comment]
Done

File: page_objects/PageCondition.js [hunk 6]
```diff
@@ -1,31 +1,39 @@
 const Page = require('./page');
 
-class condition extends Page {
-    get pasteFrame() {
+class Condition extends Page {
+    get PasteMainTextArea() {
```
Comment (rimus) [inline page_objects/PageCondition.js]: Only classes should start with Capitals. Please rename methods and fields.

File: page_objects/PageCondition.js [hunk 7]
```diff
@@ -1,31 +1,39 @@
 const Page = require('./page');
 
-class condition extends Page {
-    get pasteFrame() {
+class Condition extends Page {
```
Comment (rimus) [inline page_objects/PageCondition.js]: ConditionPage

File: page_objects/check.js [hunk 8]
```diff
@@ -1,11 +1,17 @@
 const Page = require('./page');
 
 class Check extends Page {
```
Comment (rimus) [inline page_objects/check.js]: CheckPage

File: page_objects/check.js [hunk 9]
```diff
@@ -1,11 +1,17 @@
...
-    get checkCondition(){
-        return $(`//ol[@class = "text"]`)
+    get CheckBashIfExist(){
```
Comment (rimus) [inline page_objects/check.js]: Only classes should start with Capitals. Please rename methods and fields.

**Review by rimus — CHANGES_REQUESTED** [review]

**Comment by DOnEbrala** [comment]
done

**Pull request merged**